apps/compute-cf.yml.j2

```
@@ -85,17 +95,58 @@ Resources:
         Tags:
           Name: !Ref AWS::StackName
 
+  JobQueue:
```

We'll want to keep the original name.
```suggestion
  BatchJobQueue:
```